docs/src/main/sphinx/connector/sqlserver.rst

```diff
@@ -127,6 +127,7 @@ SQL Server Type                     Trino Type
 ``bigint``                          ``bigint``
 ``smallint``                        ``smallint``
 ``int``                             ``integer``
+``tinyint``                         ``smallint``
```

Comment:
This should come before `smallint`.

Reply:
I have a question about the type mapping documentation: what is the principle of ordering, I checked some other connector's documentation and there seems to be no uniform principle.

One suggestion is: could we be consistent with the type order defined by `java.sql.Types`, I observed that the order of the types in `JdbcClient#toColumnMapping` would be consistent with the order defined in `java.sql.Types`.